mailbox/tools/copier/src/test/java/org/apache/james/mailbox/tools/copier/MailboxCopierTest.java

```diff
@@ -73,66 +73,66 @@ public void setup() throws BadCredentialsException, MailboxException {
 
         srcMemMailboxManager = newInMemoryMailboxManager();
         dstMemMailboxManager = newInMemoryMailboxManager();
-        
+
     }
-    
+
     /**
      * Feed the source MailboxManager with the number of mailboxes and
      * messages per mailbox.
-     * 
-     * Copy the mailboxes to the destination Mailbox Manager, and assert the number 
+     *
+     * Copy the mailboxes to the destination Mailbox Manager, and assert the number
      * of mailboxes and messages per mailbox is the same as in the source
      * Mailbox Manager.
-     * 
-     * @throws MailboxException 
-     * @throws IOException 
+     *
+     * @throws MailboxException
+     * @throws IOException
      */
     @Test
     public void testMailboxCopy() throws MailboxException, IOException {
         DataProvisioner.feedMailboxManager(srcMemMailboxManager);
-       
+
         assertMailboxManagerSize(srcMemMailboxManager, 1);
-        
+
         mailboxCopier.copyMailboxes(srcMemMailboxManager, dstMemMailboxManager);
         assertMailboxManagerSize(dstMemMailboxManager, 1);
-        
+
         // We copy a second time to assert existing mailboxes does not give issue.
         mailboxCopier.copyMailboxes(srcMemMailboxManager, dstMemMailboxManager);
         assertMailboxManagerSize(dstMemMailboxManager, 2);
-        
+
     }
-    
+
     /**
      * Utility method to assert the number of mailboxes and messages per mailbox
      * are the ones expected.
-     * 
-     * @throws MailboxException 
-     * @throws BadCredentialsException 
+     *
+     * @throws MailboxException
+     * @throws BadCredentialsException
      */
     private void assertMailboxManagerSize(MailboxManager mailboxManager, int multiplicationFactor) throws BadCredentialsException, MailboxException {
         MailboxSession mailboxSession = mailboxManager.createSystemSession("manager");
         mailboxManager.startProcessingRequest(mailboxSession);
 
         List<MailboxPath> mailboxPathList = mailboxManager.list(mailboxSession);
-        
+
         assertThat(mailboxPathList).hasSize(DataProvisioner.EXPECTED_MAILBOXES_COUNT);
-        
+
         for (MailboxPath mailboxPath: mailboxPathList) {
             MailboxSession userSession = mailboxManager.createSystemSession(mailboxPath.getUser());
             mailboxManager.startProcessingRequest(mailboxSession);
             MessageManager messageManager = mailboxManager.getMailbox(mailboxPath, userSession);
             assertThat(messageManager.getMetaData(false, userSession, FetchGroup.NO_UNSEEN).getMessageCount()).isEqualTo(DataProvisioner.MESSAGE_PER_MAILBOX_COUNT * multiplicationFactor);
         }
-        
+
         mailboxManager.endProcessingRequest(mailboxSession);
         mailboxManager.logout(mailboxSession, true);
-        
+
     }
-    
+
```

Comment:
Careful about unintendeed re-indents that pollutes the changeset.

We tend to put them in separated commits to ease reviewer job.

Reply:
It was not a re-indent but only an automatic space cleaning, I could remove them if you want